public/app/core/components/SVG/SanitizedSVG.tsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import SVG, { Props } from 'react-inlinesvg';
+import { v4 as uuidv4 } from 'uuid';
 
 import { textUtil } from '@grafana/data';
 
-export const SanitizedSVG = (props: Props) => {
-  return <SVG {...props} cacheRequests={true} preProcessor={getCleanSVG} />;
+import { getSvgId, getSvgStyle } from './utils';
+
+type SanitizedSVGProps = Props & { cleanStyle?: boolean };
+let shouldCleanSvgStyle = false;
+
+export const SanitizedSVG = (props: SanitizedSVGProps) => {
+  const { cleanStyle, ...inlineSvgProps } = props;
+  shouldCleanSvgStyle = cleanStyle ?? false;
```

Comment:
:( 

looking closer... this does not make sense -- we can not have a global state toggled by the individual component!

I think we need two `getCleanSVG` functions -- one that cleans and another that does not